python_agent/zmq_client.py

```
@@ -0,0 +1,93 @@
+        message = {"command_type": command_type}
+        if params:
+            message["params"] = params
+        
+        # For AI Bridge, commands are prefixed
+        if command_type == "get_observation":
+            raw_message = "command:get_observation"
+        elif command_type == "execute_action":
+            # The action_type and parameters for execute_action are wrapped in the 'params' dict
+            # which then becomes the JSON payload for "command:execute_action:"
+            if not params or "action_type" not in params:
+                raise ValueError("Missing 'action_type' in params for execute_action")
```

Ruff (0.11.9)

Avoid specifying long messages outside the exception class

(TRY003)